spec/services/idv/steps/ipp/ssn_step_spec.rb

```diff
@@ -16,7 +17,12 @@
   end
 
   let(:flow) do
-    Idv::Flows::InPersonFlow.new(controller, {}, 'idv/in_person')
+    Idv::Flows::InPersonFlow.new(controller, {}, 'idv/in_person').tap do |flow|
+      flow.flow_session = {
+        pii_from_user: {},
+        threatmetrix_session_id: threatmetrix_session_id,
```

Comment:
if this sets the session id... then what does the assertion check below? should we leave it blank here to make sure it gets assigned/